exercises/binary/description.md

```diff
@@ -19,7 +19,7 @@ as a linear combination of powers of 10:
 - The rightmost digit gets multiplied by 10^0 = 1
 - The next number gets multiplied by 10^1 = 10
 - ...
-- The *n*th number gets multiplied by 10^*(n-1)*.
+- The *n*th number gets multiplied by 10^_(n-1)_.
```

Comment:
This looks weird on both sides.

Reply:
This is _wrong_. `description.md` should not be formatted.

Reply:
But this is meant to be formatting, isn't it?

Reply:
oh. Maybe. fuck. hahaha